src/main/java/se/sundsvall/installation/api/model/InstallationsResponse.java

```diff
@@ -0,0 +1,26 @@
+package se.sundsvall.installation.api.model;
+
+import static io.swagger.v3.oas.annotations.media.Schema.AccessMode.READ_ONLY;
+
+import java.util.List;
+
+import generated.se.sundsvall.datawarehousereader.InstallationDetails;
```

Comment:
This is still using DWR classes which it shouldn't. Create own classes to decouple this service from the underlying DWR service.